02_configure_host.sh

```diff
@@ -37,7 +37,19 @@ fi
 # Allow ipmi to the virtual bmc processes that we just started
 sudo iptables -I INPUT -i virbr0 -p udp -m udp --dport 6230:6235 -j ACCEPT
 
+# Need to route traffic from the provisioning host.
+if [ -z $EXT_IF ]; then
+  sudo iptables -t nat -A POSTROUTING --out-interface ${EXT_IF} -j MASQUERADE
+  sudo iptables -A FORWARD --in-interface virbr0 -j ACCEPT
+fi
+
+# Need to pass the interface for the provisioning network on baremetal
+if [ -z $PRO_IF ]; then
+  brctl addif ${PRO_IF} brovc
```

Comment:
```suggestion
  sudo ip link set "$PRO_IF" master brovc
```
Let's use the more ubiquitous iproute2